pyglet/window/cocoa/pyglet_delegate.py

```diff
@@ -40,11 +40,13 @@
 from pyglet.libs.darwin.cocoapy import NSApplicationDidUnhideNotification
 from pyglet.libs.darwin.cocoapy import send_super, get_selector
 from pyglet.libs.darwin.cocoapy import PyObjectEncoding
+from pyglet.libs.darwin.cocoapy import quartz
 from .systemcursor import SystemCursor
 
 NSNotificationCenter = ObjCClass('NSNotificationCenter')
 NSApplication = ObjCClass('NSApplication')
 
+
```

Comment:
No big deal, but this is just a whitespace change, and doesn't belong in this commit.